### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by jainarayanjain** [amoghmadan/Table2XSV#9] [Q&A] [answered]
Is this project still active, as the latest release on PyPI says that it was last updated on January 5th, 2022.
Reference Link: https://pypi.org/project/Table2XSV/#history

**Comment by amoghmadan** [comment, marked as answer]
Yes, the project is active and [here](https://github.com/amoghmadan/Table2XSV/tree/feature/move-from-setup-to-pyproject) is the latest work done with it, you will soon see a new release.

**Reply by jainarayanjain** [reply]
Good to hear that
